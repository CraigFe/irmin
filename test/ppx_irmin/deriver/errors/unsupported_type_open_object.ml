type 'a t = < .. > [@@deriving irmin]
