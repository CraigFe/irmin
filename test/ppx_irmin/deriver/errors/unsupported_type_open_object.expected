File "unsupported_type_open_object.ml", line 1, characters 0-37:
Error: ppx_irmin: open object type < .. >  encountered. Object types must be closed.
